eox_core/api/v1/tests/test_users.py

```diff
@@ -0,0 +1,97 @@
+"""
+Test module for users viewset.
+"""
+from django.contrib.auth.models import User
+from django.test import TestCase
+from mock import patch
+from rest_framework.test import APIClient
+
+
+class UsersAPITest(TestCase):
+    """Test class for users viewset."""
+
+    patch_permissions = patch('eox_core.api.v1.permissions.EoxCoreAPIPermission.has_permission', return_value=True)
+
+    def setUp(self):
+        """Setup method for test class."""
+        self.user = User(username="test", email="test@example.com", password="testtest")
+        self.client = APIClient()
+        self.url = "/api/v1/user/"
+        self.client.force_authenticate(user=self.user)
+
+    @patch_permissions
+    @patch('eox_core.api.v1.serializers.UserSignupSource')
+    @patch('eox_core.api.v1.views.get_edxapp_user')
+    @patch('eox_core.api.v1.views.EdxappUserReadOnlySerializer')
+    def test_update_success(self, user_serializer, get_edxapp_user, signup_source, _):
+        """Used to test updating safe fields of an edxapp user."""
+        update_data = {
+            "username": "test",
+            "password": "test",
+        }
+        user_serializer.return_value.data = {}
+        get_edxapp_user.return_value = self.user
+        signup_source.objects.filter.return_value.count.return_value = 1
+
+        response = self.client.patch(self.url, data=update_data, format="json")
+
+        self.assertEqual(response.status_code, 200)
```

Comment:
the convention says that first the expected value and second the result, use status from drf instead of 200